dockerfiles/Dockerfile

```diff
@@ -221,6 +227,9 @@ RUN echo "deb http://deb.debian.org/debian bookworm-backports main" >> /etc/apt/
     libatomic1 \
     libgcrypt20 \
     libyaml-0-2 \
+    libboost-system-dev \
```

Comment:
As above, do we really need the development libraries here or can we just use the runtime ones?

Reply:
This section is unrelated to building so changing it here should have no impact on compilation.

Reply:
Yes but this is not compiling anything, this is the debug image so again should just have the minimum set of packages per the `deb-extractor` stage.